packages/google_atlas/lib/src/google_atlas_controller.dart

```diff
@@ -51,4 +51,9 @@ class GoogleAtlasController implements AtlasController {
       double north, double east, double south, double west) {
     return _controller.moveCamera(null);
   }
+
+  @override
+  void changeUserLocationIcon(String asset) {
+    // TODO: implement changeUserLocationIcon
```

Comment:
Not sure if a `TODO` makes sense here since it's not supported by Google-Maps
```suggestion
    // This functionality is not yet supported by Google-Maps.
```